main/tests/server/src/com/google/refine/util/StringUtilsTests.java

```diff
@@ -0,0 +1,20 @@
+package com.google.refine.util;
+
+import com.google.refine.RefineTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.time.OffsetDateTime;
+import com.google.refine.util.StringUtils;
+
+public class StringUtilsTests extends RefineTest {
+
+    @Test
+    public void objectToString() {
+        Object[] nums = {4, "hello", true, 0.01};
+        OffsetDateTime time = OffsetDateTime.parse("2017-01-02T01:02:03Z");
+
+        Assert.assertEquals("[4, hello, true, 0.01]", StringUtils.toString(nums));
+        Assert.assertEquals("", StringUtils.toString(null));
```

Comment:
Shouldn't this also test an Array that is empty, i.e `[]`?  Instead of a String that is empty, `""`?

Also, I am wondering about multi-dimensional arrays which is kinda the point of deepToString... If the array contains other arrays as elements, the string representation contains their contents and so on.  https://www.geeksforgeeks.org/arrays-deeptostring-in-java-with-example/